packages/webapp/pages/account/invite.tsx

```diff
@@ -103,6 +104,7 @@ const AccountInvitePage = (): ReactElement => {
           description={labels.referral.generic.inviteText}
           onNativeShare={onShareOrCopyLink}
           onClickSocial={onTrackShare}
+          shortenUrl={false}
```

Comment:
Why false?

Reply:
Because the invite URL is already shortened